src/render/painter.js

```diff
@@ -427,6 +450,8 @@ class Painter {
             coordsDescendingSymbol[id] = sourceCache.getVisibleCoordinates(true).reverse();
         }
 
+        this._updateFog();
```

Comment:
This is also called in map.js. Does it need to be called in both places?

Reply:
Yes good catch, there's an extra function call here.

Reply:
Removed in https://github.com/mapbox/mapbox-gl-js/pull/10564/commits/01d931ff39bfb8b7adab1bf539ed1c422489ccb4.